iam/keycloak/dojot-provider/README.md

```diff
@@ -194,22 +234,38 @@ $JBOSS_HOME/bin/standalone.sh --server-config=standalone-ha.xml
 
 ## Configuration
 
-| Environment variable | Default value | Description |
-| -------------------- | ------------- | ----------- |
-| DOJOT_KAFKA_SERVERS        | `kafka:9092`  | Connection address with Kafka. |
-| DOJOT_KAFKA_CLIENT_ID      | `keycloak`    | Client identifier to connect with Kafka. |
-| DOJOT_KAFKA_TOPIC          | `dojot-management.dojot.tenancy` | Kafka topic in which messages regarding the creation and removal of tenants (Realms) will be published. |
+| Environment variable | Reference values | Description |
+| -------------------- | ---------------- | ----------- |
+| DOJOT_ROOT_URL       | `http://localhost:8080`  | Root URL for accessing the dojot platform. |
+| DOJOT_KAFKA_SERVERS  | `kafka:9092`     | Connection address with Kafka. |
+| DOJOT_KAFKA_CLIENT_ID | `keycloak`      | Client identifier to connect with Kafka. |
+| DOJOT_KAFKA_TOPIC    | `dojot-management.dojot.tenancy` | Kafka topic in which messages regarding the creation and removal of tenants (Realms) will be published. |
 | DOJOT_VALID_REALM_NAME_REGEX | `^[a-zA-Z0-9]{1,30}$` | Regular expression to be applied in the name of the newly created tenant to force it to be compatible with the rules of the dojot platform. |
 | DOJOT_CUSTOM_REALM_REP_FILE | `/opt/dojot/customRealmRepresentation.json` | Full path of the json file used as a basis for customizing the tenant according to the rules of the dojot platform. |
 
-
 _Note that_ the json file must follow the structure (Schema) defined in the
 [REST API](https://www.keycloak.org/docs-api/12.0/rest-api/index.html#_partialimportrepresentation)
 of the keycloak.
 The file can be included in the keycloak container `VOLUME ["/opt/dojot/"]`.
 Check the [example](../../examples/keycloak_kafka/) directory for more details.
 
 
+## SMTP server configuration
+
+| Environment variable | Reference values | Description |
+| -------------------- | ---------------- | ----------- |
+| DOJOT_SMTP_HOST      | `smtp.gmail.com` | GMail SMTP server host example. |
```

Comment:
If the DOJOT_SMPT_ * are empty, will an error occur?

Reply:
In the `DojotProviderFactoryImpl.java` there is a method called `initSMTPServerConfig()`, If the `DOJOT_SMTP_HOST` variable is not defined or is empty, the SMTP settings will not be passed to the Keycloak.